instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientAttributesGetter.java

```diff
@@ -20,9 +20,23 @@
  */
 public interface DbClientAttributesGetter<REQUEST> extends DbClientCommonAttributesGetter<REQUEST> {
 
+  /**
+   * @deprecated Use {@link #getDbQueryText(REQUEST)} instead.
+   */
+  @Deprecated
   @Nullable
   String getStatement(REQUEST request);
 
+  @Nullable
+  String getDbQueryText(REQUEST request);
+
+  /**
+   * @deprecated Use {@link #getOperationName(REQUEST)} instead.
+   */
+  @Deprecated
   @Nullable
   String getOperation(REQUEST request);
+
+  @Nullable
+  String getOperationName(REQUEST request);
```

Comment:
we decided at some point to include the full attr name in the method name
```suggestion
  String getDbOperationName(REQUEST request);
```